src/microbe_stage/organelle_components/MovementComponent.cs

```diff
@@ -59,11 +66,13 @@ protected override void CustomAttach()
     ///   Calculate the momentum of the movement organelle based on
     ///   angle towards middle of cell
     /// </summary>
-    private static Vector3 CalculateForce(Hex pos, float momentum)
+    private static Vector3 CalculateForce(Hex pos, float momentum, Vector3 defaultPos)
```

Comment:
Is defaultPos intended to define the behavior of an organelle placed in the middle of the cell, as mimicking an organelle at this position?

I think it should be clarified in the function documentation. Especially as it does not seem obvious that this is the only right way to go

Reply:
Thats how it was ,i didnt make that change.